gstreamer-sdp/src/auto/s_d_p_message.rs

```diff
@@ -0,0 +1,396 @@
+        }
+    }
+
+    pub fn emails_len(&self) -> u32 {
+        unsafe {
+            ffi::gst_sdp_message_emails_len(self.to_glib_none().0)
+        }
+    }
+
+    //pub fn get_attribute(&self, idx: u32) -> /*Ignored*/Option<SDPAttribute> {
+    //    unsafe { TODO: call ffi::gst_sdp_message_get_attribute() }
+    //}
+
+    pub fn get_attribute_val(&self, key: &str) -> Option<String> {
+        unsafe {
+            from_glib_none(ffi::gst_sdp_message_get_attribute_val(self.to_glib_none().0, key.to_glib_none().0))
+        }
+    }
+
+    pub fn get_attribute_val_n(&self, key: &str, nth: u32) -> Option<String> {
+        unsafe {
+            from_glib_none(ffi::gst_sdp_message_get_attribute_val_n(self.to_glib_none().0, key.to_glib_none().0, nth))
+        }
+    }
+
+    //pub fn get_bandwidth(&self, idx: u32) -> /*Ignored*/Option<SDPBandwidth> {
+    //    unsafe { TODO: call ffi::gst_sdp_message_get_bandwidth() }
+    //}
+
+    //pub fn get_connection(&self) -> /*Ignored*/Option<SDPConnection> {
+    //    unsafe { TODO: call ffi::gst_sdp_message_get_connection() }
+    //}
+
+    pub fn get_email(&self, idx: u32) -> Option<String> {
+        unsafe {
+            from_glib_none(ffi::gst_sdp_message_get_email(self.to_glib_none().0, idx))
+        }
+    }
+
+    pub fn get_information(&self) -> Option<String> {
+        unsafe {
+            from_glib_none(ffi::gst_sdp_message_get_information(self.to_glib_none().0))
+        }
+    }
+
+    //pub fn get_key(&self) -> /*Ignored*/Option<SDPKey> {
+    //    unsafe { TODO: call ffi::gst_sdp_message_get_key() }
+    //}
+
+    pub fn get_media(&self, idx: u32) -> Option<SDPMedia> {
+        unsafe {
+            from_glib_none(ffi::gst_sdp_message_get_media(self.to_glib_none().0, idx))
+        }
+    }
+
+    //pub fn get_origin(&self) -> /*Ignored*/Option<SDPOrigin> {
+    //    unsafe { TODO: call ffi::gst_sdp_message_get_origin() }
+    //}
+
+    pub fn get_phone(&self, idx: u32) -> Option<String> {
+        unsafe {
+            from_glib_none(ffi::gst_sdp_message_get_phone(self.to_glib_none().0, idx))
+        }
+    }
+
+    pub fn get_session_name(&self) -> Option<String> {
+        unsafe {
+            from_glib_none(ffi::gst_sdp_message_get_session_name(self.to_glib_none().0))
+        }
+    }
+
+    //pub fn get_time(&self, idx: u32) -> /*Ignored*/Option<SDPTime> {
+    //    unsafe { TODO: call ffi::gst_sdp_message_get_time() }
+    //}
+
+    pub fn get_uri(&self) -> Option<String> {
+        unsafe {
+            from_glib_none(ffi::gst_sdp_message_get_uri(self.to_glib_none().0))
+        }
+    }
+
+    pub fn get_version(&self) -> Option<String> {
+        unsafe {
+            from_glib_none(ffi::gst_sdp_message_get_version(self.to_glib_none().0))
+        }
+    }
+
+    //pub fn get_zone(&self, idx: u32) -> /*Ignored*/Option<SDPZone> {
+    //    unsafe { TODO: call ffi::gst_sdp_message_get_zone() }
+    //}
+
+    pub fn init(&mut self) -> SDPResult {
```

Comment:
This makes not much sense, we rather need a proper constructor somehow

Reply:
Not generated for now, but I've got a patch that fixes this